src/pages/Contributing/GitConventions/index.md

```diff
@@ -1,4 +1,113 @@
 ---
 title: 'Git conventions'
-order: 10
----
\ No newline at end of file
+order: 5
+---
+
+## Create and name a new branch
+
+Each branch name consists of a `type`, an optionnal issue `id` and a `description` (_Definitions are available further down the page_)
+
+```shell
+git checkout -b type-id-description
+```
+
+> MOZAIC Team use **JIRA as their project management tool** <br> BU's use **Github issues to manage their tasks into MOZAIC**
+
+
+---
+
+
+## Create and write a commit message
```

Comment:
just `commit message : `
and I think we should start with this part and the conventional commit stuffs

Reply:
Actually, i would like to keep the actual order because it's the way we work, we first create a new branch before creating a commit. If i read Thomas's feedback, i supposed that we need something very didactics like step 1, step 2 and so on. Especially when we decide to put all those information into the same page